### ISSUE TITLE
[utf8] some emojis trigger invalid UTF8 string 

### ISSUE THREAD

**Issue by atomrc** [StableLib/stablelib#66]
When trying to encode strings containing emojis, some emojis seem to trigger errors. 

a simple reproducer would be the following:

```js
utf8.encode('❤️') // Uncaught Error: utf8: invalid string
```

Any idea if there is a workaround I could use? 
Could this be fixed at the library level? 

Thanks :)

**Comment by atomrc** [comment]
As a side note, this works. So I do believe @stablelib/utf-8 `encode` function should be able to handle that payload :) 

```js
    const enc = new TextEncoder().encode('❤️');
    const dec = decode(enc);
```

**Comment by dchest** [comment]
Thank you! Fixed in 1.0.2.

**Comment by atomrc** [comment]
Thank you @dchest for such a fast fix 🙏
